js/main.js

```diff
@@ -1 +1 @@
-console.log("Hello World")
+console.log('Hello World')
```

Comment:
What is this file doing here?

Reply:
It's been here from the beginning, all project has his own main.js file.